From c6cb447e2318ee662d7f582e114ee4607fd7b3f8 Mon Sep 17 00:00:00 2001
From: Patrick Roberts <patrick@silentbluesystems.com>
Date: Mon, 26 Nov 2012 22:29:50 -0800
Subject: [PATCH 2/3] Tweak keyboardmapping for slate

---
 Src/base/KeyboardMapping.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Src/base/KeyboardMapping.h b/Src/base/KeyboardMapping.h
index 1af5942..040d2bf 100644
--- a/Src/base/KeyboardMapping.h
+++ b/Src/base/KeyboardMapping.h
@@ -26,7 +26,7 @@
 
 #include <Qt>
 
-#if (defined(TARGET_DESKTOP) || defined(TARGET_EMULATOR))
+#if (defined(TARGET_DESKTOP) || defined(TARGET_EMULATOR)) || defined(TARGET_X86_TABLET)
 
 typedef struct {
         int devicekey;
-- 
1.7.4.4

