diff --git a/LunaSysMgr.upstart b/LunaSysMgr.upstart
index 8993204..0170188 100644
--- a/LunaSysMgr.upstart
+++ b/LunaSysMgr.upstart
@@ -10,7 +10,7 @@ respawn
 pre-start script
        echo "1" > /proc/sys/vm/overcommit_memory
        mkdir -p /var/luna/preferences
-       if grep -qs "qemu" /etc/hostname ; then
+       if grep -qs "slate" /etc/hostname ; then
                touch /var/luna/preferences/ran-first-use
        fi
        if [ ! -f /var/luna/preferences/ran-first-use ]