diff --git a/LunaSysMgr.upstart b/LunaSysMgr.upstart
index c197d80..4dd6b31 100644
--- a/LunaSysMgr.upstart
+++ b/LunaSysMgr.upstart
@@ -11,7 +11,7 @@ pre-start script
        echo "1" > /proc/sys/vm/overcommit_memory
        mkdir -p /var/luna/preferences
        mkdir -p /var/luna/data
-       if grep -qs "qemu" /etc/hostname ; then
+       if grep -qs "slate" /etc/hostname ; then
                touch /var/luna/preferences/ran-first-use
        fi
        if [ ! -f /var/luna/preferences/ran-first-use ]