From d7f6a41ddcb51b1d62e43c8dfa46d207c702724a Mon Sep 17 00:00:00 2001
From: Patrick Roberts <patrick@silentbluesystems.com>
Date: Sat, 8 Dec 2012 14:06:01 -0800
Subject: [PATCH] Disable vsync on when not using qws

---
 Src/base/DisplayManager.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Src/base/DisplayManager.cpp b/Src/base/DisplayManager.cpp
index 0ee11bf..1d2bb1a 100644
--- a/Src/base/DisplayManager.cpp
+++ b/Src/base/DisplayManager.cpp
@@ -3596,7 +3596,7 @@ void DisplayManager::changeVsyncControl(bool enable)
 	if (s_forceVsyncDisable)
 		return;
 
-#if defined(TARGET_DEVICE) && defined(HAVE_OPENGL)
+#if defined(TARGET_DEVICE) && defined(HAVE_OPENGL) && defined(HAVE_QWS)
 	QGLContext* gc = (QGLContext*) QGLContext::currentContext();
 	if (gc) {
 		setEglSwapInterval(enable ? 1 : 0);
-- 
1.7.4.4

