From a7c27be4bd8c21ded18fd814ae0fc9dff7ad4a65 Mon Sep 17 00:00:00 2001
From: Patrick Roberts <patrick@silentbluesystems.com>
Date: Wed, 3 Oct 2012 12:32:34 -0700
Subject: [PATCH] Disable webos platform

---
 src/plugins/platforms/platforms.pro |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/plugins/platforms/platforms.pro b/src/plugins/platforms/platforms.pro
index 6bf8675..c7de126 100644
--- a/src/plugins/platforms/platforms.pro
+++ b/src/plugins/platforms/platforms.pro
@@ -13,6 +13,8 @@ webos {
     } else:taskone {
         SUBDIRS += taskone
         SUBDIRS += palm
+    } else:slate {
+        SUBDIRS += palm eglfs
     } else {
         SUBDIRS += palm
         SUBDIRS += webos
-- 
1.7.4.4


