diff --git a/configure.ac b/configure.ac
index 0caa1b1..92a0e52 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1402,6 +1402,10 @@ if test "x$enable_egl" = xyes; then
 
     AC_CHECK_FUNC(mincore, [DEFINES="$DEFINES -DHAVE_MINCORE"])
 
+    if test "$have_x" != yes; then
+        DEFINES="$DEFINES -DMESA_EGL_NO_X11_HEADERS"
+    fi
+
     if test "$enable_static" != yes; then
         # build egl_glx when libGL is built
         if test "x$enable_glx" = xyes; then
-- 
1.7.3.4
